wormhole-connect/src/config/types.ts

```diff
@@ -75,6 +75,20 @@ export interface BridgeDefaults {
   requiredNetwork?: ChainName;
 }
 
+export interface ExtendedTransferDetails extends TransferDetails {
+  fromWallet: string;
```

Comment:
we use "wallet" to refer to the extension used (phantom, etc) in a different telemetry event. so can we use `walletAddress` here?